hu.irl.keystep37-settings.yml

```diff
@@ -0,0 +1,33 @@
+app-id: hu.irl.keystep37-settings
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: keystep37-settings
+finish-args:
+  # show windows with Wayland
+  - --socket=wayland
+  # show windows using X11, if Wayland is not available
+  - --socket=fallback-x11
+  # share IPC namespace with the host (necessary for X11)
+  - --share=ipc
+  # MIDI
+  - --device=all
```

Comment:
if all you need is MIDI then the pulseaudio permission should be used. (and `dri` likely)

Reply:
Fixed, thanks.